php/plan.sh

```diff
@@ -18,19 +18,24 @@ pkg_deps=(
   core/libjpeg-turbo
   core/libpng
   core/openssl
+  core/readline
   core/zlib
 )
 pkg_build_deps=(
   core/bison2
   core/gcc
   core/make
   core/re2c
-  core/readline
 )
 pkg_bin_dirs=(bin sbin)
 pkg_lib_dirs=(lib)
 pkg_include_dirs=(include)
 pkg_interpreters=(bin/php)
+pkg_exports=(
+  [port]=port
+  [listen]=listen
```

Comment:
What do you think about replacing the 'listen' export for a 'local_only' boolean export? The problem I am trying to solve is when the service is bound to localhost only. In that case the dependent service cannot connect with the IP address ('bind.php.first.sys.ip') it has to use 127.0.0.1. I have another PR [here](https://github.com/habitat-sh/core-plans/pull/1242) where I am trying to add the 'local_only' flag for mysql.